Tasks/UseRubyVersionV0/userubyversion.ts

```diff
@@ -41,11 +41,7 @@ export async function useRubyVersion(parameters: TaskParameters, platform: Platf
     if (platform !== Platform.Windows) {
         // replace the default
         const dest: string = '/usr/bin/ruby';
-        if (fs.existsSync(dest)) {
-            task.debug('removing ' + dest);
```

Comment:
minor: Is it useful to still check for the pre-existence and log a debug message, even if `-f` handles either case?